packages/cli/src/cmds/validator/handler.ts

```diff
@@ -169,10 +171,15 @@ export async function validatorHandler(args: IValidatorCliArgs & GlobalArgs): Pr
       distributed: args.distributed,
     },
     metrics
-  );
+  ).catch((err) => {
```

Comment:
As mentioned in https://github.com/ChainSafe/lodestar/pull/5511#discussion_r1200304394 by @tuyennhv we shouldn't use await and catch.

Reply:
I earlier used the `try/catch` pattern as suggested by @tuyennhv. You asked it does not feel good wrap it. So alternative is the following approach. As we don't prefer using `let` in our code.

Reply:
Yeah, all of the proposed solutions look sub-optimal